opamp/observiq/testdata/latest/quickupdater

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
For the windows binaries, do we have the code that generated the binaries? I think it would be helpful to have the committed to know what generated them if it isn't too much trouble, even if it's dead simple (I assume they are just the same as these bash scripts).

Reply:
@BinaryFissionGames that's fair. so you're thinking just putting the go files in the testdata/latest directory?
```
package main

import "time"

func main() {
	time.Sleep(1 * time.Second)
}
```

Reply:
Yeah, I'm thinking just to commit the go files.

Reply:
Is it possible to do this with a batch script or powershell script and avoid a go binary? Do we need it to be an `.exe`?

Reply:
@cpheps I initially tried a powershell script. but i couldn't execute the scripts directly like with a shell script. i have to execute powershell.exe instead. maybe there is some straightforward way to do this and get a script to run, but it was easier for me to just make a couple of small windows exe's instead...